src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -55,16 +74,45 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   // Send a user to datastore. Update the current information about the user if one exists.
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userGitHubID = request.getParameter("githubID");
-    if (userGitHubID.trim().isEmpty()) {
-      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Invalid ID");
-      return;
+    String userGitHubAuthToken = request.getParameter("gitHubAuthToken");
```

Comment:
nit: In the future it might be worth checking to see if this is null before using it with the GitHub interface. Omitting the auth token should most likely raise a different error than if the user is supplying an auth token that is invalid.

Reply:
sure thing!

Reply:
sure thing!